cms/djangoapps/contentstore/views/item.py

```diff
@@ -266,6 +269,8 @@ def service(self, block, service_name):
                 return DjangoXBlockUserService(self._user)
             if service_name == "studio_user_permissions":
                 return StudioPermissionsService(self._user)
+            if service_name == "lti-configuration":
+                return ConfigurationService(LTIConsumerHideFieldsFlag)
```

Comment:
We're merging this PR as it with a TODO of this ticket https://openedx.atlassian.net/browse/EDUCATOR-121 to support Django in LTI xBlock which is necessary to move this code out of edx-platform. At this point, i'm not sure how much work will it need to support Django in an xBlock. Probably, i've to create an Django app and add it to the installed apps in edx-platform. Do we have any existing example xBlock which was refactored to support Django ?

@nasthagiri  @bradenmacdonald @cpennington 

@scottrish FYI ^^